### PR TITLE
Fix dotnet5

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Need merge https://github.com/neo-project/neo/pull/2179 first.

### CODE REVIEW

**Review by shargon — APPROVED** [review]

File: src/RpcServer/RpcServer.csproj [hunk 1]
```diff
@@ -11,9 +11,5 @@
       <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
     </None>
   </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.ResponseCompression" Version="2.2.0" />
```
Comment (shargon) [inline src/RpcServer/RpcServer.csproj:16]: Did you test the compression?

Comment (superboyiii) [reply src/RpcServer/RpcServer.csproj:16]: > Did you test the compression?

Not deeply, but tested RPC APIs(getblockcount, getblock, etc) work well.

Comment (shargon) [reply src/RpcServer/RpcServer.csproj:16]: Could you check if the response it's compressed?

Comment (superboyiii) [reply src/RpcServer/RpcServer.csproj:16]: > Could you check if the response it's compressed?

image
Looks like compressed.

Comment (shargon) [reply src/RpcServer/RpcServer.csproj:16]: Good!

**Review by shargon — APPROVED** [review]

**Review by vncoelho — DISMISSED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**